src/MatrixClient.ts

```diff
@@ -667,6 +668,7 @@ export class MatrixClient extends EventEmitter {
 
     /**
      * Gets the joined members in a room. The client must be in the room to make this request.
+     * This request is usually faster than getRoomMembers.
```

Comment:
This isn't relevant to the API structure and describes an implementation detail - please remove it.